### PR TITLE
BZ1878360 Clarified cluster system.

### PR DESCRIPTION
**Pull request by jonquilwilliams**
https://bugzilla.redhat.com/show_bug.cgi?id=1878360

QE & peer review required.

CP to 4.5 and 4.6

Thanks!

### CODE REVIEW

File: networking/enable-cluster-wide-proxy.adoc [hunk 1]
```diff
@@ -14,7 +14,7 @@ The cluster-wide proxy is only supported if you used a user-provisioned infrastr
 
 == Prerequisites
 
-* Review the xref:../installing/install_config/configuring-firewall.adoc#configuring-firewall[sites that your cluster requires access to] and determine whether any of them must bypass the proxy. By default, all cluster egress traffic is proxied, including calls to the cloud provider API for the cloud that hosts your cluster. Add sites to the Proxy object's `spec.noProxy` field to bypass the proxy if necessary.
+* Review the xref:../installing/install_config/configuring-firewall.adoc#configuring-firewall[sites that your cluster requires access to] and determine whether any of them must bypass the proxy. By default, all cluster system egress traffic is proxied, including calls to the cloud provider API for the cloud that hosts your cluster. (System-wide proxy affects system components only, not user workloads.) Add sites to the Proxy object's `spec.noProxy` field to bypass the proxy if necessary.
```
Comment (ahardin-rh) [inline networking/enable-cluster-wide-proxy.adoc:17]: We try to avoid parentheticals, so I would remove the parentheses. Other than that, looks great!

**Comment by jonquilwilliams** [comment]
Peer review edits made and committed. Waiting on QA.

**Comment by jonquilwilliams** [comment]
@zhaozhanqi Please do QE review. https://bugzilla.redhat.com/show_bug.cgi?id=1878360 Thank you!

**Pull request merged**